08LessonEight/in-class-project/css/homework-style.css

```diff
@@ -0,0 +1,88 @@
+body {
+  margin: 0 auto;
+  width: 690px;
+  font-family: Arial, sans-serif;
+  font-size: 12px;
+  line-height:14px;
+}
+header {
+  text-align: center;
+  width: 400px;
+  margin: 0 auto;
+}
+main {
+
+    margin: 0 auto;
```

Comment:
If i remove this entire CSS rule it wont change a thing. What where you trying to do here?